src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -72,6 +72,32 @@ public class BQConnection implements Connection {
   /** Boolean to determine whether or not to use legacy sql (default: false) * */
   private final boolean useLegacySql;
 
+  /**
+   * Enum that describes whether to create a job in projects that support stateless queries. Copied
+   * from BigQueryImpl
+   */
+  public static enum JobCreationMode {
```

Comment:
Can we add a link here to the API docs or maybe even a permalink to the BigQuery code itself that has this?

Reply:
My preference is code; linking to permalink should stay since the BigQuery SDK is a public Github repo?